### PR TITLE
Add more russian models

### PR DESCRIPTION
**Pull request by Samoed**
## Checklist

- [X] Run tests locally to make sure nothing is broken using `make test`. 
- [X] Run the formatter to format the code using `make lint`. 

### Adding datasets checklist

**Reason for dataset addition**: ... 

- [X] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb -m {model_name} -t {task_name}` command.
  - deepvk/USER-base
  - deepvk/deberta-v1-base
  - DeepPavlov/rubert-base-cased
  - DeepPavlov/distilrubert-small-cased-conversational
  - DeepPavlov/rubert-base-cased-sentence
  - cointegrated/LaBSE-en-ru
  - sergeyzh/rubert-tiny-turbo
  - sergeyzh/LaBSE-ru-turbo
- [X] Run tests locally to make sure nothing is broken using `make test`. 
- [X] Run the formatter to format the code using `make lint`. 

### Adding a model checklist

 - [x] I have filled out the ModelMeta object to the extent possible
 - [x] I have ensured that my model can be loaded using
   - [x] `mteb.get_model(model_name, revision_id)` and
   - [x] `mteb.get_model_meta(model_name, revision_id)`
 - [x] I have tested the implementation works on a representative set of tasks.

### CODE REVIEW

File: docs/mmteb/points.md
Comment (KennethEnevoldsen) [inline docs/mmteb/points.md:1]: Can you revert the change to the table - it makes differences very hard to track. Adding new lines should be sufficient

**Review by KennethEnevoldsen — COMMENTED** [review]
@Muennighoff where is the best place to submit these results?

File: docs/mmteb/points.md
Comment (Samoed) [reply docs/mmteb/points.md:1]: Yes, I can, but specifically format the table because it is hard to read in plain text mode and a bit broken. I can make it in different PR.

Comment (KennethEnevoldsen) [reply docs/mmteb/points.md:1]: A seperate PR would be reasonable

**Comment by Muennighoff** [comment]
I think right now it is still https://huggingface.co/datasets/mteb/results? cc @orionw

**Comment by orionw** [comment]
> I think right now it is still https://huggingface.co/datasets/mteb/results? cc @orionw

I haven't move the folder yet, so there is great for now.  I'll try to get to it soon!

**Comment by KennethEnevoldsen** [comment]
Perfect - I will merge this in for the model implementations - but the results will probably be moved.
